generate_rust.sh

```diff
@@ -39,17 +37,22 @@ if ! cmd_exists protoc-gen-gofast; then
     done
 fi
 
-protoc -I.:${GOGO_ROOT}:${GOGO_ROOT}/protobuf --rust_out ../src *.proto || ret=$?
-
-echo "extern crate protobuf;" > ../src/lib.rs
-for file in `ls *.proto`
+protoc -I.:${GOGO_ROOT}:${GOGO_ROOT}/protobuf --rust_out ../src *.proto || exit $?
+protoc -I.:${GOGO_ROOT}:${GOGO_ROOT}/protobuf --rust-grpc_out ../src *.proto || exit $?
+pop
+
+push src
+rm -f lib.rs
+echo "extern crate protobuf;" > lib
```

Comment:
I think it is ok. We need to remove line `mod lib` in `lib.rs` if we keep `lib.rs` .